src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -219,7 +219,7 @@ export class Sen extends Analyser {
 
 		let lastSenState = this.lastSenState
 
-		if ((lastSenState == null) || (lastSenState.isDone === true)) {
+		if ((typeof lastSenState === 'undefined') || (lastSenState.isDone === true)) {
```

Comment:
Why?

Reply:
I merged in a copy of code that didn't get the updates from my last PR it appears.
That is Why

Reply:
fixed